RPG/Src/XCOM2RPGOverhaul/Classes/X2TemplateHelper_RPGOverhaul.uc

```diff
@@ -482,20 +490,28 @@ static function PatchWeapons()
 						WeaponTemplate.NumUpgradeSlots = default.DefaultWeaponUpgradeSlots;
 						break;
 					case 'shotgun':
+						if (default.bAddWeaponsChanges)
```

Comment:
bAddWeaponsChanges is not very precise in communicating what it does.
It sounds like it would disable all weapon changes but e.g slots are still patched and stuff lik HeavyWeaponMobilityPenalty and Suppressions is still added.
I suggest naming the variable 
 `bPatchShotguns`
 and using that only for the shotgun changes.